examples/drivenet/drivenet_train.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to DriveNet\n",
+    "\n",
+    "DriveNet is a DNN able to drive an agent on a specific route. Among others, DriveNet can do lane-following while respecting traffic lights.\n",
+    "\n",
+    "### Imitation Learning and Its limitations\n",
+    "DriveNet is trained using imitation learning on our Lyft L5 Prediction Dataset 2020. We feed examples of real driving experiences to the model and expect it to take the same actions as the driver did in those episodes. This is very similar to how tasks like classification are usually solved.\n",
+    "\n",
+    "Imitiation Learning is powerful, but it has a strong limitation. It's not trivial for a trained model to generalise well on out-of-distribution data.\n",
+    "\n",
+    "After training DriveNet, we would like it to take full control and drive the AV in an autoregressive fashion (i.e. by following its own predictions).\n",
+    "\n",
+    "In this scenario it's very easy for errors to compound and make the AV drift away from the original distribution. During training our DriveNet has seen good examples of driving only. In particular, this means **almost perfect midlane following**. However, even a small constant displacement during evaluate can accumulate enough error to lead the AV completely out of its distribution in a matter of seconds.\n",
+    "\n",
+    "![drifting](../../images/drivenet/drifting.svg)\n",
```

Comment:
VERY NICE!
What is the little top part coming in though, is it a lane?
It looks too narrow for a lane and is confusing.